mintpy/objects/stack.py

```diff
@@ -506,6 +508,48 @@ def get_design_matrix4step_func(date_list, step_date_list):
 
             return A
 
+
+        def get_design_matrix4exp_func(date_list, exp_list):
+            """design matrix/function model of exponential postseismic relaxation estimation
+            Parameters: date_list      : list of dates in YYYYMMDD format
+                        exp_list[0]    : Exp func(s) onset time(s) with date in YYYYMMDD
+                        exp_list[1]    : tau(s), characteristic time(s) in decimal days
```

Comment:
How about changing the arguments here to `exp_onset_times, exp_char_times` explicitly instead of `exp_list`?

Same comment for the log func as well.

Reply:
No problem. I think this will be better. Thanks.